src/plutus/builtin.md

```diff
@@ -91,3 +91,41 @@ No concrete syntax is provided for values of type `𝚋𝚕𝚜𝟷𝟸_𝟹𝟾
 It is not possible to parse such values, and they will appear as `(con bls12_381_mlresult <opaque>)` if output by a program.
 
 ## Built-in Functions
+
+Plutus includes a set of built-in functions.
+Like in Haskell, these can be polymorphic, accepting both type and term arguments.
+However, Plutus distinguishes between two kinds of type arguments:
+- A built-in-polymorphic type argument (denoted as `𝑎#`, `𝑏#` etc. in the spec) is limited to built-in types.
```

Comment:
I wonder if we can just write `a` instead of `a#` for built-in polymorphic type variables now?  We used to have `a*` and `a#` but now `a*` has been replaced by `*` so maybe we don't need the `#`.  I left it in the  specification because if we ever add a specification of PIR or TPLC we'll need two different kinds of type variables again (eg, in the typed languages the branches of `ifThenElse` have to have the same type but in UPLC they don't (and terms don't have types in UPLC anyway).

Reply:
I think we likely can. However this section is a guidance of how to read the spec so it should use what the spec uses.